README.md

```diff
@@ -0,0 +1,116 @@
+# Monolog: A Note-Taking CLI Tool
+
+Monolog is a simple command-line application for managing notes. It allows you to add notes, view today's notes, and retrieve the most recent notes.
+
+## Features
+
+- Add Notes: Add new notes quickly from the command line.
+- Today's Notes: Display notes created today.
+- Recent Notes: View a specific number of the most recent notes.
+
+## Prerequisites
+
+Ensure you have the following tools installed:
+
+### 1. Rust and Cargo
+
+Run `rustc --version` and `cargo --version` to confirm installation.
+
+### 2. SQLite: SQLite is required for the database.
+
+## Installation
+
+Follow these steps to install and build monolog:
+
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/alexpetrean80/monolog.git
```

Comment:
I would mention the single line install `cargo install --git ...` as most users (well this doesn't have any users but me but still) will not want to write Rust code so having the repo locally is a useless step